content/scripts/check-pr-links.rb

```diff
@@ -35,6 +42,19 @@
 
 # takes a `URI` object, returns [ok, html-or-error]
 def check_link(url)
+  # Ignore non-web protocols like mailto:
+  unless url.scheme == 'https' || url.scheme == 'http'
+    return [true, 'Not an HTTP(S) URL']
+  end
```

Comment:
```suggestion
   return [true, 'Not an HTTP(S) URL'] unless url.scheme == 'https' || url.scheme == 'http'
```
This makes it more clear to me but just a thought

Reply:
Works for me!